things/api.py

```diff
@@ -677,9 +677,53 @@ def token(**kwargs) -> Union[str, None]:
     return database.get_url_scheme_auth_token()
 
 
-def link(uuid):
-    """Return a things:///show?id=uuid link."""
-    return f"things:///show?id={uuid}"
+def url(uuid=None, command="show", **kwargs) -> str:
+    """
+    Return a things:///<command>?id=uuid&… url.
+
+    For details about available commands and their parameters
+    consult the Things URL Scheme documentation
+    [here](https://culturedcode.com/things/help/url-scheme/).
+
+    Parameters
+    ----------
+    uuid : str, optional
+        A valid uuid of any Things object
+        Defaults to None
```

Comment:
See `tasks()` for how we currently document defaults. The way we do it is not set in stone, but I think it's helpful if a consistent style is used.

Reply:
Will fix